recipes/clingo/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+if [ -z "${PYTHON}" ]; then
+    PYTHON="$(which python)"
+fi
+
+cmake .. \
+    -DCMAKE_CXX_COMPILER="${CXX}" \
+    -DCMAKE_C_COMPILER="${CC}" \
+    -DPYTHON_EXECUTABLE="${PYTHON}" \
+    -DCLINGO_REQUIRE_PYTHON=ON \
+    -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
+    -DPYCLINGO_USER_INSTALL=OFF \
+    -DCLINGO_BUILD_WITH_LUA=OFF \
+    -DCLINGO_MANAGE_RPATH=OFF \
```

Comment:
As the issue is with rpaths, have you tried this:
```suggestion
    -DCLINGO_MANAGE_RPATH=ON \
```

Reply:
I think this is not the problem. There is the following line in the cmake configure output:
```
Found Python: /opt/conda/bin/python3.7 (found version "3.7.6") found components: Interpreter Development
```
This is outside the build environment! Should be easy to fix.